ARGO/src/PickUpManager.cpp

```diff
@@ -17,7 +17,7 @@ void PickUpManager::init(SDL_Renderer* t_renderer)
 	PickUpFactory* factory = new PickUpFactory(m_renderer);
 	for (int i = 0; i < PICKUP_POOL_SIZE; i++)
 	{
-		factory->createDrop(1, m_pickUps[i]);
+		factory->createDrop(2, m_pickUps[i]);
```

Comment:
why the change here?

Reply:
Health Is Implemented and affects the player and Hud. This is here to demonstrate that both systems are working.